### PR TITLE
Build v 2.7.5:

### PR DESCRIPTION
**Pull request by akabanovs**
- Update build scripts.
 - Update scripts that outputs the path to mathjax.
 - update recipe with m2-bash for win.

### CODE REVIEW

File: recipe/build.sh [hunk 1]
```diff
@@ -2,27 +2,25 @@
...
-
 
-mkdir -p "${bin_dir}"
-cp "${RECIPE_DIR}/mathjax-path" "${bin_dir}/"
-if [[ "$OSTYPE" == "msys" ]]; then
-    cp "${RECIPE_DIR}/mathjax-path.bat" "${bin_dir}/"
-else
-    chmod +x "${bin_dir}/mathjax-path"
-fi
+mv config "$mathjax/"
```
Comment (JeanChristopheMorinPerso) [inline recipe/build.sh]: Wouldn't it be simpler to just delete what needs to be deleted adn then copy/move everything else (i.e. `cp -r . "${mathjax}/"`)?

So basically
```
rm -f *.sh LICENSE *.md .gitignore .travis.yml bower.json composer.json latest.js package.json

mv . "$PREFIX/lib/mathjax"

mkdir -p "$PREFIX/bin"
cp "${RECIPE_DIR}/mathjax-path" "$PREFIX/bin/"
chmod +x "$PREFIX/bin/mathjax-path"
```

Comment (akabanovs) [reply recipe/build.sh]: At the current location there are also conda related scripts such as conda_build.sh for example. I don't want the script to remove/move itself. I also think it is nice to see what it is actually copying/moving.

**Review by skupr-anaconda — APPROVED** [review]
LGTM

File: recipe/build.sh [hunk 2]
```diff
@@ -2,27 +2,27 @@
...
-
-mkdir -p "${bin_dir}"
-cp "${RECIPE_DIR}/mathjax-path" "${bin_dir}/"
-if [[ "$OSTYPE" == "msys" ]]; then
-    cp "${RECIPE_DIR}/mathjax-path.bat" "${bin_dir}/"
-else
-    chmod +x "${bin_dir}/mathjax-path"
-fi
+mathjax="$PREFIX/lib/mathjax"
+mkdir -p "$mathjax" || exit 1
```
Comment (JeanChristopheMorinPerso) [inline recipe/build.sh]: If you set `set -e` at the top of the file, you won't need to do `|| exit. 1`.

File: recipe/build.sh [hunk 3]
```diff
@@ -2,27 +2,27 @@
...
+cp -r fonts "$mathjax/" || exit 1
+rm -rf fonts
+mv jax "$mathjax/" || exit 1
+mv localization "$mathjax/" || exit 1
+mv test "$mathjax/" || exit 1
+mv unpacked "$mathjax/" || exit 1
+
+rm *.md ".gitignore" ".travis.yml" "bower.json" "composer.json" "latest.js" "package.json" || exit 1
+cwd="$(pwd)"
+cp -r "$cwd/." "$mathjax/"
```
Comment (JeanChristopheMorinPerso) [inline recipe/build.sh]: You say that you don't want to move the whole directory because it has the conda script. Isn't this doing exactly that?

Comment (akabanovs) [reply recipe/build.sh]: Well, it is copying, not moving. Then it removes the sh files inside. And all the directories had been already moved so -r is not actually needed.
If you want it that way, okay. I just thought that moving is less resource-intensive than copying.

Comment (akabanovs) [reply recipe/build.sh]: I actually tried to robocopy everything on a prefect build for windows yesterday, and it is then when the windows build got stuck.

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Pull request merged**